modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/item/selector/test/ContentDashboardItemSubtypeItemSelectorViewTest.java

```diff
@@ -310,7 +310,7 @@ public void testGetDataWithDDMStructuresFromTheGuestGroup()
 				ddmStructure.getStructureId(),
 				itemSubtypeJSONObject.getLong("classPK"));
 			Assert.assertEquals(
-				"Test Structure (Liferay DXP)",
+				"Test Structure (Liferay)",
```

Comment:
Could you use ddmStructure.getName() or something like that

Reply:
Hi @cgoncas, 
ddmStructure.getName() returns the whole xml structure as follows: 

<?xml version='1.0' encoding='UTF-8'?><root available-locales="en_US" default-locale="en_US"><Name language-id="en_US">Test Structure</Name></root>

Internally, to that Name defined there, we're adding the site (Liferay in this example) so we cannot easily match it to an specific field in the ddmStructure

Reply:
Yeahh, but this test is going to fail if you run the profile dxp. Because the name of the site in this case is Liferay DXP.

Reply:
Let me give it a thought...

Reply:
What do you think if we twisted like this?

			```
Assert.assertTrue(
				itemSubtypeJSONObject.getString("label")
					.contains(ddmStructure.getName(LocaleUtil.US)));
```

We still test the correctness of the name but we ignore the site as it can change

Reply:
I think this is a valid approach